### ISSUE TITLE
[java] CloseResource: False positive for FileSystems.getDefault()

### ISSUE THREAD

**Issue by SaschaRiemer** [pmd/pmd#5067]
**Affects PMD Version:**
7.2.0

**Rule:**
CloseResource
https://docs.pmd-code.org/latest/pmd_rules_java_errorprone.html#closeresource

**Description:**
Instances of FileSystems.getDefault() trigger that rule. As per the javadoc for java.nio.file.FileSystem: `The default file system cannot be closed.` Trying to close the default file system would throw an UnsupportedOperationException.

**Code Sample demonstrating the issue:**

```java
public class Example {
    public static void main(String[] args) {
        final var fs = FileSystems.getDefault();
    }
}
```

**Expected outcome:**

PMD reports a violation at line 3, but that's wrong. That's a false positive.

**Running PMD through:** *Gradle*
